core/drivers/crypto/hisilicon/hisi_qm.c

```diff
@@ -24,6 +24,10 @@
 #define QM_MB_OP_SHIFT		14
 #define QM_MB_OP_WR		0
 #define QM_MB_OP_RD		1
+#define QM_MB_STATUS_MASK GENMASK_32(12, 9)
+#define QM_MB_WAIT_READY_CNT 10
+#define QM_MB_WAIT_MAX_CNT 21000
+#define QM_MB_WAIT_PERIOD 200
```

Comment:
I suggest you add some tabulation indent to values for consistency with the macro definitions below and above.